offices/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+
+from offices.views import OfficeView
+
+urlpatterns = [
+    path('/<int:id>', OfficeView.as_view()),
```

Comment:
보다 명확하게 적어주시면 좋을 것 같습니다!
```suggestion
    path('/<int:building_id>', OfficeView.as_view()),
```